examples/error-cause/error_cause.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates error cause in Ballerina.
```

Comment:
```suggestion
description: This BBE demonstrates the error causes in Ballerina.
```
Is it `how error causes are identified`? What is demonstrated by the BBE is not clear.

Reply:
So ballerina facilitates having a cause in an error value so the bbe shows how to add a cause to the error and access the cause

Reply:
@suleka96 - Then, shall we say `This BBE demonstrates how to add a cause to the error and access it.`?